gpkit/tests/t_geometric_program.py

```diff
@@ -91,9 +91,15 @@ def test_Mddtest(self):
         self.assertEqual(gp2.A, CootMatrix(row=[0, 1],
                                            col=[0, 0],
                                            data=[-1, 1]))
-        self.assertEqual(gp3.A, CootMatrix(row=[0, 2, 3],
-                                           col=[0, 0, 0],
-                                           data=[-1, 1, -1]))
+        # order of variables with a posynomial is not stable
+        #   (though monomial order is)
```

Comment:
Hmm, we could make CootMatrix accept those arguments.

Monomial terms are stable, yeah.